content/post/67-crdt-research-meetup.md

```diff
@@ -0,0 +1,60 @@
+---
+date: 2019-01-28
+url: 67-crdt-research-meetup
+tags: crdt research meetup videos ipfs
+title: 2018 CRDT Research Meetup - Lisbon
+author: Pedro Teixeira
+---
+
+On April last year some members of the IPFS team gathered in Lisbon for a week of fun hack. During this week we mostly hacked and collaborated on some aspects of IPFS, libp2p and applications built on top of them, but also took one day to host a CRDT Research meetup.
+
```

Comment:
Add here a small banner with screenshots of the presenters. Similar to the one found in https://blog.ipfs.io/72-ann-ipfs-camp/ but with people instead of places.

Reply:
do we already have approved images for these people? If not, I'd suggest we move forward without the back-and-forth required to source images/approvals